loadgen/cmd/otelbench/.chloggen/loadgenreceiver-tsdb-millis.yaml

```diff
@@ -0,0 +1,18 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
not a bug in loadgenreceiver in a strict sense, but millisecond precision is just something not supported by many metric backends. It makes sense to workaround it for us (at least for now), so that otelsoak and otelbench don't need additional transformprocessor workaround to add a random dimension for TSDB.